.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: required
 
 language: go
+go_import_path: github.com/uber/jaeger
```

Comment:
just so I'm on the same boat, why is this required?

Reply:
after the repo move the github path is going to be different, but all the imports in the code refer to uber/jaeger, so it won't  build. This instruction tells Travis to relocate the repo to this path under GOPATH before building.